crates/bevy_core_pipeline/src/motion_blur/mod.rs

```diff
@@ -0,0 +1,161 @@
+//! Per-object, per-pixel motion blur.
+//!
+//! Add the [`MotionBlurBundle`] to a camera to enable motion blur. See [`MotionBlur`] for more
+//! documentation.
+
+use crate::core_3d;
+use bevy_app::{App, Plugin};
+use bevy_asset::{load_internal_asset, Handle};
+use bevy_ecs::{bundle::Bundle, component::Component, schedule::IntoSystemConfigs};
+use bevy_render::{
+    extract_component::{ExtractComponent, ExtractComponentPlugin, UniformComponentPlugin},
+    render_graph::{RenderGraphApp, ViewNodeRunner},
+    render_resource::{Shader, ShaderType, SpecializedRenderPipelines},
+    renderer::RenderDevice,
+    Render, RenderApp, RenderSet,
+};
+
+pub mod node;
+pub mod pipeline;
+
+/// Adds [`MotionBlur`] and the required depth and motion vector prepasses to a camera entity.
+#[derive(Bundle, Default)]
+pub struct MotionBlurBundle {
+    pub motion_blur: MotionBlur,
+    pub depth_prepass: crate::prepass::DepthPrepass,
```

Comment:
Nit: Import prepass types

Reply:
I don't know what you are asking for here. As far as I can tell this matches all the other render bundles.

Reply:
Instead of doing crate:prepass::Foo, just import Foo directly.